docs/authentication/guides/jaas.md

```diff
@@ -68,3 +68,69 @@ datahub-frontend-react:
 ```
 
 And then run `docker-compose up` against your compose file.
+
+## Adding New Users via user.props File
+
+:::note Important
+Adding users via the `user.props` will require disabling existence checks on GMS using the `METADATA_SERVICE_AUTH_ENFORCE_EXISTENCE_ENABLED=false` environment variable or using the API to enable the user prior to login.
+The directions below demonstrate using the API to enable the user.
+:::
+
+To define a set of username / password combinations that should be allowed to log in to DataHub (in addition to the root 'datahub' user),
+create a new file called `user.props` at the file path `${HOME}/.datahub/plugins/frontend/auth/user.props` within the `datahub-frontend-react` container
+or pod.
+
+This file should contain username:password specifications, with one on each line. For example, to create 2 new users,
+with usernames "janesmith" and "johndoe", we would define the following file:
+
+```
+// custom user.props
+janesmith:janespassword
+johndoe:johnspassword
```

Comment:
In what cases should this be done? We should provide guidance that 9 times out of 10, users shouldn't be added this way.